src/containers/my-resources/categories-container/CategoriesContainer.tsx

```diff
@@ -49,25 +43,51 @@ const CategoriesContainer = () => {
   const breakpoints = useBreakpoints()
   const { page, handleChangePage } = usePagination()
   const { openModal, closeModal } = useModalContext()
-  const dispatch = useAppDispatch()
   const [selectedItemId, setSelectedItemId] = useState<string>('')
   const [updateResourceCategory] = useUpdateResourceCategoryMutation()
+  const { handleErrorAlert } = useSnackbarAlert()
+  const queryClient = useQueryClient()
+  const dispatch = useAppDispatch()
 
   const { sort } = sortOptions
   const itemsPerPage = getScreenBasedLimit(breakpoints, itemsLoadLimit)
 
-  const onResponseError = useCallback(
-    (error?: ErrorResponse) => {
-      dispatch(
-        openAlert({
-          severity: snackbarVariants.error,
-          message: getErrorKey(error)
-        })
-      )
-    },
-    [dispatch]
+  const getCategories = useCallback(
+    () =>
+      ResourceService.getResourcesCategories({
+        limit: itemsPerPage,
+        skip: (page - 1) * itemsPerPage,
+        sort,
+        name: searchTitle.current
+      }),
+    [page, itemsPerPage, sort, searchTitle]
+  )
+
+  const deleteCategory = useCallback(
+    (id?: string) => ResourceService.deleteResourceCategory(id ?? ''),
+    []
   )
 
+  const {
+    error,
+    data: categories,
+    isLoading,
+    refetch: fetchData
+  } = useQuery({
+    queryFn: getCategories,
+    queryKey: ['categories', page, itemsPerPage, sort, searchTitle]
+  })
+
+  const updateInfo = useCallback(async () => {
+    await queryClient.invalidateQueries({ queryKey: ['categories'] })
+  }, [fetchData])
```

Comment:
You have an eslint error, fix it please
```suggestion
  const updateInfo = useCallback(async () => {
    await queryClient.invalidateQueries({ queryKey: ['categories'] })
  }, [queryClient])
```